src/comparator/src/index.ts

```diff
@@ -95,7 +95,7 @@ type RevisionStringFormatter = (cell: RevisionCell) => string;
 type RevisionDeltaStringFormatter = (cell: RevisionDeltaCell) => string;
 type TotalStringFormatter = (cell: TotalCell, sizeKey: string) => string;
 type DeltaStringFormatter = (cell: DeltaCell | TotalDeltaCell, sizeKey: string) => string;
-type ArtifactFilter = (row: ArtifactRow) => boolean;
+export type ArtifactFilter = (row: ArtifactRow) => boolean;
```

Comment:
I want to export this so I can annotate my function.

You might want to export other types too, but this was sufficient for my needs.

Reply:
Can you export the others as well while we're  here? At least this block of items and the `FormattingOptions` below